woocommerce/woocommerce
# Remove docs plugin from monorepo

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:



This PR removes the docs plugin from the monorepo, as it's not actively being developed here, and one of its dependencies caused issues when attempting to install / build with php 8.3.



Closes # .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Verify that all references to the docs plugin are removed
2. Try `pnpm i && pnpm build`
3. Make sure CI passes



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 
This PR fixes a linting error and does not require a change entry

</details>
